hack/lib/common.sh

```diff
@@ -876,4 +876,19 @@ function kube::common::generate_kubeproxy_certs {
     fi
 }
 
-
+function kube::common::wait-until-mizar-ready {
+    echo "Waiting for Mizar CRDs to reach 'Provisioned' state ..."
+    until ${KUBECTL} --kubeconfig "${CERT_DIR}/admin.kubeconfig" get vpcs | grep Provisioned; do
+        sleep 5
+    done
+    until ${KUBECTL} --kubeconfig "${CERT_DIR}/admin.kubeconfig" get dividers | grep Provisioned; do
+        sleep 5
+    done
+    until ${KUBECTL} --kubeconfig "${CERT_DIR}/admin.kubeconfig" get bouncers | grep Provisioned; do
+        sleep 5
+    done
+    until ${KUBECTL} --kubeconfig "${CERT_DIR}/admin.kubeconfig" get subnets | grep Provisioned; do
+        sleep 5
+    done
+    echo "Mizar is ready."
```

Comment:
Please add message "Cluster is ready to use ..."

Reply:
**Did adjustment for message. Now typical screen log in scale up:**
Kata Setup done.
*******************************************
Waiting for Mizar CRDs to reach 'Provisioned' state
................
Mizar is ready.
Local Kubernetes cluster is running. Press Ctrl-C to shut it down.

Logs:
  /tmp/kube-apiserver0.log
  /tmp/kube-controller-manager.log


  /tmp/kube-proxy.log
  /tmp/kube-scheduler.log
  /tmp/kubelet.log

To start using your cluster, you can open up another terminal/tab and run:

  export KUBECONFIG=/var/run/kubernetes/admin.kubeconfig
Or
  export KUBECONFIG=/var/run/kubernetes/adminN(N=0,1,...).kubeconfig

  cluster/kubectl.sh

Alternatively, you can write to the default kubeconfig:

  export KUBERNETES_PROVIDER=local

  cluster/kubectl.sh config set-cluster local --server=https://ip-172-31-26-226:6443 --certificate-authority=/var/run/kubernetes/server-ca.crt
  cluster/kubectl.sh config set-credentials myself --client-key=/var/run/kubernetes/client-admin.key --client-certificate=/var/run/kubernetes/client-admin.crt
  cluster/kubectl.sh config set-context local --cluster=local --user=myself
  cluster/kubectl.sh config use-context local
  cluster/kubectl.sh

**Typical screen log in scale out:**
Arktos Setup done.
Waiting for Mizar CRDs to reach 'Provisioned' state
...........
Mizar is ready.
Local Kubernetes cluster is running. Press Ctrl-C to shut it down.

Logs:
  /tmp/kube-apiserver0.log
  /tmp/kube-controller-manager.log


  /tmp/kube-scheduler.log


To start using your cluster, you can open up another terminal/tab and run:

  export KUBECONFIG=/var/run/kubernetes/admin.kubeconfig
Or
  export KUBECONFIG=/var/run/kubernetes/adminN(N=0,1,...).kubeconfig

  cluster/kubectl.sh

Alternatively, you can write to the default kubeconfig:

  export KUBERNETES_PROVIDER=local

  cluster/kubectl.sh config set-cluster local --server=https://ip-172-31-21-250:6443 --certificate-authority=/var/run/kubernetes/server-ca.crt
  cluster/kubectl.sh config set-credentials myself --client-key=/var/run/kubernetes/client-admin.key --client-certificate=/var/run/kubernetes/client-admin.crt
  cluster/kubectl.sh config set-context local --cluster=local --user=myself
  cluster/kubectl.sh config use-context local
  cluster/kubectl.sh
Tenant Partition Cluster is Running ...